composer.json

```diff
@@ -105,6 +105,11 @@
             "**/Tests/"
         ]
     },
+    "autoload-dev": {
+        "psr-4": {
+            "Symfony\\Component\\Console\\Tests\\": "src/Symfony/Component/Console/Tests"
+        }
+    },
```

Comment:
That's what this section of composer is for.

Do you prefer manual including in every test where class is needed?